controllers/suite_util_test.go

```diff
@@ -482,12 +488,12 @@ func waitSecretCreated(resourceKey types.NamespacedName) {
 	}, timeout, interval).Should(BeTrue())
 }
 
-func ensureSecretNotCreated(resourceKey types.NamespacedName) {
+func waitSecretGone(resourceKey types.NamespacedName) {
```

Comment:
I'm afraid you've broken all the places where `ensureSecretNotCreated` was used. I'd create another method instead.

Reply:
ok done